buddyboss/buddyboss-platform
# Fixed-Forums total topic count is not consistent

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2473 .

### How to test the changes in this Pull Request:

1. Go to the BuddyBoss > Forums > Discussions
2. Select any Discussions and set them as a Type: Super Sticky
3. switch the theme to Twenty-Twenty.
4. go to the forums listing page and check the count of discussions.

### Proof Screenshots or Video

https://www.loom.com/share/2622ee6e20144d2c92876062f85b03d5

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
